frontend/lib/src/components/core/Block/styled-components.ts

```diff
@@ -88,6 +88,16 @@ export const StyledElementContainer = styled.div<StyledElementContainerProps>(
       marginTop: "-0.375rem",
       marginBottom: "-0.375rem",
     },
+    // Lower the min height of stacked/grouped checkboxes to have them appear visually
+    // closer together to each other.
+    // To detect & cover all grouped/stacked checkboxes, we apply a complex CSS selector
+    // that selects all checkboxes that are directly followed by another checkbox.
+    // Since the last checkbox in a group isn't followed by another checkbox, we also
+    // need to target the direct sibling (if it is a checkbox) of any of the targeted checkboxes.
```

Comment:
thanks for the detailed comment! 

nit: since this is a somewhat more complex selector, it might make sense to provide an example of what is selected and what is not,e.g.
```html
// Examples:
// not selected because single checkbox:
// <div><checkbox /></div>
// selected except for the last one:
// <div><checkbox /><checkbox /><checkbox /></div>
// not selected because not directly followed:
// <div><checkbox /><div>some-text</div><checkbox /></div>
```

And perhaps also write the query in the comment without the variables which might make it easier to read due to the missing brackets and symbols:
`&:hash(& > check) > check, &:has(...`

Reply:
Also, does this has an effect on checkboxes if they are displayed in a row instead of in a column and if yes, is that a problem?

Reply:
> Also, does this has an effect on checkboxes if they are displayed in a row instead of in a column and if yes, is that a problem?

Nope, since it would have a different hierarchical structure that is not selected.

Reply:
I added a couple of examples, but a bit more higher level since the actual DOM structure is more complicated.